test/python/algorithms/eigensolvers/test_vqd.py

```diff
@@ -70,7 +68,7 @@ def setUp(self):
         self.fidelity = ComputeUncompute(Sampler())
         self.betas = [50, 50]
 
-    @data(H2_PAULI, H2_OP, H2_SPARSE_PAULI)
+    @data(H2_PAULI, H2_SPARSE_PAULI)
```

Comment:
I am not too happy with not testing `Operator` here. What we do in these cases is not remove the test that raises deprecation warnings, but assert that the warnings are raised when using deprecated classes. This is to confirm that the deprecation warning is raised where it is expected (and the motivation behind making tests fail with deprecation warnings).

In this case, you would need to do a type check of "`op`" to make sure that you are only asserting the warning when it is of type `Operator`.